src/template.ts

```diff
@@ -62,7 +62,10 @@ export const compileSchema = (schemaName: string, typeName: string) =>
 export const validateFn = (
   typeName: string,
   schemaName: string,
-) => `const rawValidate${typeName} = ${compileSchema(schemaName, typeName)};
+) => `export const rawValidate${typeName} = ${compileSchema(
```

Comment:
I wonder about renaming this to `is{typeName}` before we export it and lock in the function name as part of the public API.